lib/audited/auditor.rb

```diff
@@ -202,7 +204,13 @@ def audit_create
       def audit_update
         unless (changes = audited_changes).empty? && audit_comment.blank?
           write_audit(action: 'update', audited_changes: changes,
-                      comment: audit_comment)
+                      comment: audit_comment) do
+            if keep_audits && audits.count > keep_audits.to_i
```

Comment:
Shouldn't this check be done inside `write_audit` function always so it runs on destroy audits as well? It would also reduce the complexity.